vendor/vendor-android/adam/src/main/kotlin/com/malinskiy/marathon/android/adam/AdamDeviceProvider.kt

```diff
@@ -155,6 +155,7 @@ class AdamDeviceProvider(
             deviceEventsChannel.cancel()
         }
         logcatManager.close()
+        client.socketFactory.close()
```

Comment:
Should stop test runs from freezing: vertx creates daemon threads and we have to close them